2-mandatory/2-water-bottle.js

```diff
@@ -63,7 +63,7 @@ and see if your answer matches the expected result at the bottom :)
 */
 
 /* ======= TESTS - DO NOT MODIFY =====
-- To run the tests for this exercise, run `npm test -- --testPathPattern 4-water-bottle.js`
+- To run the tests for this exercise, run `npm test -- --testPathPattern 2-water-bottle.js`
```

Comment:
You know what you should do when you've found a bug in the question? Create a real PR with the correction and ask for a review with the authors!
Otherwise it's out of the scope of your commit, so don't change it here.